docs/sources/flow/reference/components/prometheus.exporter.mysql.md

```diff
@@ -0,0 +1,191 @@
+[mysqld_exporter](https://github.com/prometheus/mysqld_exporter) for collecting stats from a MySQL server.
+
+## Usage
+
+```river
+prometheus.exporter.mysql "LABEL" {
+    data_source_name = "DATA SOURCE NAME"
+}
+```
+
+
+## Arguments
+The following arguments can be used to configure the exporter's behavior.
+All arguments are optional. Omitted fields take their default values.
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`data_source_name`                                | `secret`       | [Data Source Name](https://github.com/go-sql-driver/mysql#dsn-data-source-name) for the MySQL server to connect to. | | yes
+`enable_collectors`                               | `list(string)` | A list of [collectors][] to enable on top of the default set.  | | no
+`disable_collectors`                              | `list(string)` | A list of [collectors][] to disable from the default set. | | no
+`set_collectors`                                  | `list(string)` | A list of [collectors][] to run. Fully overrides the default set. | | no
+`lock_wait_timeout`                               | `int`          | Timeout, in seconds, to acquire a metadata lock. | `2`| no
+`log_slow_filter`                                 | `bool`         | Used to avoid queries from scrapes being logged in the slow query log. | `false` | no
+
+
+Set a `lock_wait_timeout` on the connection to avoid potentially long wait times for metadata locks. View more detailed documentation on `lock_wait_timeout` [in the MySQL documentation](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_lock_wait_timeout).
+
+> **NOTE**: `log_slow_filter` is not supported by Oracle MySQL.
+
+
+[collectors]: #supported-collectors
+
+## Blocks
+
+The following blocks are supported inside the definiton of
+`prometheus.exporter.mysql` to configure collector-specific options:
+
+Hierarchy | Name | Description | Required
+--------- | ---- | ----------- | --------
+info_schema.processlist      | [info_schema.processlist][] | Configures the `info_schema.processlist` collector. | no
+info_schema.tables           | [info_schema.tables][] | Configures the `info_schema.tables` collector. | no
+perf_schema.eventsstatements | [perf_schema.eventsstatements][] | Configures the `perf_schema.eventsstatements` collector. | no
+perf_schema.file_instances   | [perf_schema.file_instances][] | Configures the `perf_schema.file_instances` collector. | no
+heartbeat                    | [heartbeat][] | Configures the `heartbeat` collector. | no
+mysql.user                   | [mysql.user][] | Configures the `mysql.user` collector. | no
+
+[info_schema.processlist]: #info_schema.processlist-block
+[info_schema.tables]: #info_schema.tables-block
+[perf_schema.eventsstatements]: #perf_schema.eventsstatements-block
+[perf_schema.file_instances]: #perf_schema.file_instances-block
+[heartbeat]: #heartbeat-block
+[mysql.user]: #mysql.user-block
```

Comment:
I tried this out with `make docs` in the `docs/` folder; the `.` gets removed from the anchor and gets replaced with no character. 

```suggestion
[info_schema.processlist]: #info_schemaprocesslist-block
[info_schema.tables]: #info_schematables-block
[perf_schema.eventsstatements]: #perf_schemaeventsstatements-block
[perf_schema.file_instances]: #perf_schemafile_instances-block
[heartbeat]: #heartbeat-block
[mysql.user]: #mysqluser-block
```